trollflow2/launcher.py

```diff
@@ -54,26 +54,52 @@
 DEFAULT_PRIORITY = 999
 
 
-def run(prod_list, topics=None):
+def get_test_message(test_message_file):
+    """Read file and retrieve the test message"""
+
+    msg = None
+    if test_message_file:
+        with open(test_message_file) as fpt:
+            msg = fpt.readline().strip('\n')
+
+    return msg
+
+
+def run(prod_list, topics=None, test_message=None):
+
+    tmessage = get_test_message(test_message)
+    if tmessage:
+        from threading import Thread as Process
```

Comment:
Any specific reason to use `Threading` instead of `multiprocessing` for "offline" mode?

Reply:
Otherwise I can't use pdb